WithSecure/withsecure-elements/_meta/smart-descriptions.json

```diff
@@ -37,7 +37,26 @@
     ]
   },
   {
-    "value": "Microsoft antimalware blocked execution of {{withsecure.amsi.content_name}} on {{host.hostname}}",
+    "value": "Executable file {file.path} blocked on {host.hostname}",
```

Comment:
For applicationControl, file.path is not extracted. I suggest using `process.executable` instead.